src/stories/Carousel.stories.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
```

Comment:
Can you update the name of the file accordingly to the component. FadedCarousel

Reply:
At the beginning I renamed it to `FadedCarousel`, but ended up adding the `withFade` option to the original `Carousel`. The code added looked fairly straightforward, what do you think?